frontend/src/components/Field/Attachment/Attachment.tsx

```diff
@@ -138,14 +143,33 @@ export const Attachment = forwardRef<AttachmentProps, 'div'>(
           }
         }
 
+        // Compress images that are too large.
+        if (
+          IMAGE_UPLOAD_TYPES_TO_COMPRESS.includes(acceptedFile.type) &&
+          maxSize &&
+          acceptedFile.size > maxSize
+        ) {
+          return imageCompression(acceptedFile, {
+            maxSizeMB: maxSize ? maxSize / MB : undefined,
+            maxWidthOrHeight: 1024,
```

Comment:
I realize 1024 is parity with angular, but 1024 is just too crap 😑😰 . Let's go with 1920 (FHD), and see if complaints die down.

Reply:
[c92cefa](https://github.com/opengovsg/FormSG/pull/5287/commits/c92cefa1bd992d578f125583db18eb5c7bf3524b)